contentful/export-processor/content-types/section.js

```diff
@@ -152,6 +152,119 @@ export class Section {
     return minimumPathsForRecommendations;
   }
 
+  /**
+   * Calcutes paths to select each answer to each question
+   */
+
+    getPathsForAllAnswers() {
```

Comment:
I'd like to see this method refactored if possible. It's quite large, and does a lot, so I think it could be improved in terms of _readability_ quite a lot.